Packs/CofenseTriage/Integrations/CofenseTriage/CofenseTriage_test.py

```diff
@@ -0,0 +1,380 @@
+import pytest
```

Comment:
your unit tests did not run in the build for some reason, will dig into it.
we support unit test in the following format: https://xsoar.pan.dev/docs/integrations/unit-testing
so only a file, in the path: `CofenseTriage/Integrations/CofenseTriage/CofenseTriage_test`
this also connects to the idea of probably having only 1 class.

Reply:
please move this tests to the `CofenseTriagev2` dir, and name it `CofenseTriagev2_test.py`
same for the other test files.